nuxeo-runtime/nuxeo-runtime-datasource/src/test/java/org/nuxeo/runtime/datasource/TestValidateConnection.java

```diff
@@ -141,8 +141,7 @@ public void testPooled(String name, CaughtSite site) throws ReportException, SQL
 
     protected void checkPooledConnection(PooledDataSource ds) throws ReportException, InterruptedException {
         TransactionHelper.startTransaction();
-        try {
-            Connection connection = ds.getConnection();
+        try (Connection connection = ds.getConnection()) {
             try (Statement statement = connection.createStatement()) {
```

Comment:
I see, in this case auto-close feature is not needed, you can leave the code as it was, and maybe add a comment why you don't auto-close the connection.
At least don't auto-close connection, it's weird to have it there and close it manually.